plugins/module_utils/azure_rm_common.py

```diff
@@ -272,8 +272,8 @@ def default_api_version(self):
     from msrestazure import AzureConfiguration
     from msrest.authentication import Authentication
     from azure.mgmt.resource.locks import ManagementLockClient
-    from azure.mgmt.recoveryservicesbackup import RecoveryServicesBackupClient
-    import azure.mgmt.recoveryservicesbackup.models as RecoveryServicesBackupModels
+    from azure.mgmt.recoveryservicesbackup.activestamp import RecoveryServicesBackupClient
```

Comment:
@GaretJax  The current version used is azure-mgmt-recoveryservicesbackup==3.0.0, and later versions (such as 7.0.0) migrate to 'ctivestamp。